core/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/model.scala

```diff
@@ -136,10 +136,15 @@ package model {
     sink: SinkConfig,
     buffer: BufferConfig
   )
-  final case class PrometheusMetricsConfig(
-    enabled: Boolean,
-    durationBucketsInSeconds: Option[List[Double]]
+
+  final case class StatsdConfig(
+    hostname: String,
+    port: Int,
+    period: FiniteDuration = 1.minute,
+    prefix: String         = "snowplow.collector"
   )
+  final case class MetricsConfig(enabled: Boolean, statsd: StatsdConfig)
```

Comment:
I wonder if `enabled` should be a sub-field of `statsd`.  Just in case we ever want a different type of metrics to enable/disable separately.

For example, many of our snowplow applications have a `metrics` block with both `stdout` and `statsd` options and they can be enabled independently e.g. [here in rdb loader](https://github.com/snowplow/snowplow-rdb-loader/blob/3.0.0/config/redshift.config.reference.hocon#L80-L98).